rust/libnewsboat/src/utils.rs

```diff
@@ -104,16 +104,17 @@ mod tests {
         assert_eq!(to_u(String::from("zero"), 1), 1);
     }
 
+    #[test]
```

Comment:
Oh shoot, I feel so stupid for missing that!

It's time for me to figure out how to check test coverage for Rust code. With C++, these "non-tests" would've triggered a warning from Coveralls.

Reply:
I missed it too. The problem stems from me not realizing I wasn't running the rust tests. As soon as I did the compiler yelled at me.